cmd/kubectl-gadget/top/top.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const DefaultStatsTTL = 2
```

Comment:
The old stats should not be only shown when there are no new stats at all, but mixed with the new stats of other nodes. A concrete use case is the `top ebpf` where you measure the cumulative times of program runs. In case one node doesn't report in time, it would not magically make the not-updated entries disappear (removing the whole node from the output) but keep them for another interval.
Maybe we have to differentiate between the use cases...

Reply:
Mmmm ok, with the example of the `top ebpf` I got that you want to combine new and old stats on purpose so that the number of entries is always the same, not just to avoid the "empty screen" (no new stats at all). I am not sure, let me think about it.

@mauriciovasquezbernal @alban any comment?

Reply:
I think it is fine like this for the normal "top" view.

But I am not sure about the "json" view. I wonder if we should give the raw output in the json view. Wdyt?

Also, I wonder about the benchmark use case where users run the benchmark for 10 seconds with `--timeout 10` and look at the final "cumul*" columns. If the last stats happen to be late, the benchmark will show cumul* values with 10% less. For this specific use case, we need to wait for the stats instead. Not sure how to fix that properly.

Reply:
I think there is not perfect solution for this issue. Maybe an alternative could be to introduce a sequence number on the data sent from each node and print all the stats from the same sequence number together? This would cause some delay on the client, as it would need to wait for all nodes, but maybe this works and avoids cases when old/new stats are reported together?